text/0000-clarify-api-namespace.md

```diff
@@ -0,0 +1,59 @@
+- Start Date: 2015-08-10
+- RFC PR: (leave this empty)
+- Refract Issue: (leave this empty)
+
+# Summary
+Clarify API description namespace and Iron out issues which rose during API Blueprint serialization to API description namespace.
+
+# Motivation
+The purpose of this document is to clear inconsistencies in the format and make it easier to build tools around it.
+
+# Detailed design
+
+## HREF Variables
+We need to clarify about the value of the `content` in the `hrefVariables` element.
+
+## Data Structure
+We don't need a `Data Structure` because an element which is an `object`, `extend` or `array` can be considered as a data structure element.
+
+*(The current spec is a lot confusing. What is the `Data Structure Element`? Why is `Data Structure` inheriting from it?)*
+
+## Resource
+Since `hrefVariables` is an element, we should move it to be under `content` rather than leave it under `attributes`.
```

Comment:
:-1: No, it is an attribute to a resource's `href` not a content of a resource in any sense (in similar fashion you would have to move `href` there too). 

Just because something was defined as an element, it does not mean you cannot use it in attributes (ref our offline discussion about whether attributes may contain elements).

See 

> When attributes is an object, it MAY contain elements itself. The element definition SHOULD be used when interacting with attributes and its properites and values.

in https://github.com/refractproject/refract-spec/blob/master/refract-spec.md#base-element